### PR TITLE
Fixed style and complexity issues

### PR DESCRIPTION
**Pull request by elpic**
Fixed issues found by rubocop.

* Increased the Quality GPA

### CODE REVIEW

File: lib/sidekiq-scheduler.rb [hunk 1]
```diff
@@ -8,10 +8,10 @@
   config.on(:startup) do
     enabled = Sidekiq::Scheduler.enabled
     scheduler_options = {
-      :scheduler => config.options.fetch(:scheduler, true),
-      :dynamic => config.options.fetch(:dynamic, false),
-      :enabled => enabled.nil? ? true : enabled,
-      :schedule => config.options.fetch(:schedule, nil)
+      scheduler: config.options.fetch(:scheduler, true),
+      dynamic:   config.options.fetch(:dynamic, false),
+      enabled:   enabled.nil? ? true : enabled,
```
Comment (snmgian) [inline lib/sidekiq-scheduler.rb]: enabled || false

Comment (elpic) [inline lib/sidekiq-scheduler.rb]: This is a option that was added recently, and no all configuration have this option, that is way if the option is not present we assume that the scheduler is enabled.

Comment (snmgian) [inline lib/sidekiq-scheduler.rb]: Other config options are read though config option, shouldn't be better to follow the same approach?

Comment (elpic) [inline lib/sidekiq-scheduler.rb]: Yes that is better. 

Done

**Pull request closed without merging**